utilities/constants.py

```diff
@@ -1,34 +1,65 @@
-# Serving Runtime
+# TrustyAI
+TRUSTYAI_SERVICE = "trustyai-service"
+TRUSTYAI_SERVICE_IMAGE = "quay.io/trustyaiservice/trustyai-service"
+MM_PAYLOAD_PROCESSORS = "MM_PAYLOAD_PROCESSORS"
+
+# TrustyAI Generic Endpoints
+TRUSTYAI_NAMES_ENDPOINT = "/info/names"
+TRUSTYAI_MODEL_METADATA_ENDPOINT = "/info"
+TRUSTYAI_UPLOAD_ENDPOINT = "/data/upload"
+
+# TrustyAI Metrics Endpoints
+TRUSTYAI_SCHEDULE = "request"
+TRUSTYAI_METRICS = "metrics"
+
+# TrustyAI Bias Metrics Endpoints
+TRUSTYAI_SPD_ENDPOINT = f"{TRUSTYAI_METRICS}/group/fairness/spd"
+
+# TrustyAI Drift Metrics Endpoints
+TRUSTYAI_METRICS_DRIFT = f"{TRUSTYAI_METRICS}/drift"
+TRUSTYAI_MEANSHIFT = "meanshift"
+TRUSTYAI_FOURIERMMD = "fouriermmd"
+TRUSTYAI_KSTEST = "kstest"
+TRUSTYAI_APPROXKSTEST = "approxkstest"
+
+TRUSTYAI_MEANSHIFT_ENDPOINT = f"/{TRUSTYAI_METRICS_DRIFT}/{TRUSTYAI_MEANSHIFT}"
```

Comment:
Could these constants be in a separate module, e.g. `utilities.constants.endpoints.metrics`?
As the number of endpoints (metrics, explainers, etc) grows in the test this might become too large.

Reply:
Agreed. I moved most of the constants to the place where they're used, and simplify how I was handling the metrics and related endpoints. If we end up having a lot of constants here that are used in several places, we can create specific modules.